plugin/lspconfig.lua

```diff
@@ -97,12 +97,24 @@ if vim.version.ge(vim.version(), { 0, 11, 2 }) then
   end
 
   api.nvim_create_user_command('LspStart', function(info)
-    if vim.lsp.config[info.args] == nil then
-      vim.notify(("Invalid server name '%s'"):format(info.args))
-      return
+    local servers = info.fargs
+
+    -- Default to enabling all servers matching the filetype of the current buffer.
+    -- This assumes that they've been explicitly configured through `vim.lsp.config`,
+    -- otherwise they won't be present in the private `vim.lsp.config._configs` table.
+    if #servers == 0 then
+      local filetype = vim.bo.filetype
+      for name, _ in pairs(vim.lsp.config._configs) do
+        local filetypes = vim.lsp.config[name].filetypes
+        if filetypes and vim.tbl_contains(filetypes, filetype) then
+          table.insert(servers, name)
+        end
+      end
     end
 
-    vim.lsp.enable(info.args)
+    for _, name in ipairs(servers) do
+      vim.lsp.enable(name)
+    end
```

Comment:
enable() accepts a list. this is a case where "configs" might be less confusing than "servers". we are enabling configs, which may eventually start a server. we aren't really enabling "servers"

```suggestion
    vim.lsp.enable(servers)
```